van-gogh-second-page/dalite/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8" />
+  <title>Van Gogh Museum</title>
+  <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+    <header>
+        <div>
+            <nav>
+                <ul class="navigation">
+                    <li><a href="#">Home</a></li>
+                    <li><a href="#">Tickets & Visit</a></li>
+                    <li><a href="#">Art & Stories</a></li>
+                    <li><a href="#">About</a></li>
+                    <li><a href="#">Webstore</a></li>
+                    <li><a href="#">NL|EN</a></li>
+                    <li><a href="#">Search</a></li>
+                </ul>
+            </nav>
+        </div>
+        <div>
+            <p class="logo">Van <br> Gogh <br> Museum <span>Amsterdam</span></p>
+        </div>
+    </header>
+    <main>
```

Comment:
I would remove paddings right and left for each child tag and set up common padding on the main so that padding is the same for each child)